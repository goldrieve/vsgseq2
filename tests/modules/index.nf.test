nextflow_process {

    name "Test Process INDEX"
    script "modules/index.nf"
    process "INDEX"

    test("Should index fasta file") {

        when {
            params {
                cores = 1
            }
            process {
                """
                input[0] = "$projectDir/tests/data/VSGome.fasta"
                input[1] = params.cores

                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
