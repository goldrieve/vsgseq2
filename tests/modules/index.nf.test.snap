{
    "Should index fasta file": {
        "content": [
            "reflengths.bin:md5,b97672b4b08d3c6eb62413304a03a2f8"
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "25.04.6"
        },
        "timestamp": "2025-08-30T11:48:14.109233035"
    }
}